config.inc.php.dist

```diff
@@ -6,12 +6,12 @@ define('USER', 'guest');
 define('PASS', 'guest');
 define('VHOST', '');
 
-define('MEDIA_NAME', '');
+define('MEDIA_NAME', 'mtt');
 define('MEDIA_STORAGE_PATH', '');
```

Comment:
'/tmp/...' perhaps confusing, as it needs to be exact same path as your mediamanager config path